src/index.js

```diff
@@ -527,6 +527,7 @@ function createConfig(options, entry, format, writeMeta) {
 					nodeResolve({
 						mainFields: ['module', 'jsnext', 'main'],
 						browser: options.target !== 'node',
+						extensions: ['.mjs', '.js', '.jsx', '.json', '.node'],
```

Comment:
just kinda wondering if there could be problems with typescript, probably not because everything in node modules should already be compiled

Reply:
I just add `jsx` to the list of [the default values](https://github.com/rollup/plugins/blob/a66c8c70fc18f3967783b4e3e16e0aecd437d1f1/packages/node-resolve/src/index.js#L25), nothing should be different for TypeScript.
:)